packages/web-core/src/services/WebAuthnService.ts

```diff
@@ -84,9 +84,17 @@ export class WebAuthnService {
     return window.PublicKeyCredential && (await window.PublicKeyCredential.isConditionalMediationAvailable());
   }
 
-  static async canUseBluetooth(): Promise<boolean> {
+  static async canUseBluetooth(): Promise<boolean | undefined> {
     try {
-      const availability = await navigator.bluetooth.getAvailability();
+      const bluetooth = navigator.bluetooth;
+
+      // When using Safari and Firefox navigator.bluetooth returns undefined => we will return undefined
+      if (!bluetooth) {
+        return;
```

Comment:
This makes sense, I think we should also return undefined in case of an error, right? (see l.105)